workflows/promote-job.ts

```diff
@@ -0,0 +1,104 @@
+import fs from 'fs-extra';
+import {cyan, green, red} from 'kleur';
+import {createPullRequest} from 'octokit-plugin-create-pull-request';
+import {Octokit as BaseOctokit} from '@octokit/rest';
+
+const Octokit = BaseOctokit.plugin(createPullRequest);
+const octokit = new Octokit({auth: process.env.GITHUB_TOKEN});
```

Comment:
Can these be combined into 1 line? having BaseOctokit and Octokit and octokit is confusing to read

Reply:
Actually I dislike having another npm package when you can easily create a PR with octokit/rest https://octokit.github.io/rest.js/v18#pulls-create

Reply:
The plugin lets me create a PR with file changes. The method you linked only creates a PR, but that would mean I would have to modify the file inline, create a commit, push a branch, and create a PR manually - the plugin consolidates all of these into a single command

fwiw this plugin was created by [gr2m](https://github.com/gr2m) who is the maintainer of the Octokit JS repo, so if we're  using Octokit it's probably fine to use this plugin.

wdyt?


As for consolidating the two lines together, this is what it might end up looking like. Not sure it's any cleaner...
```suggestion
import {Octokit} from '@octokit/rest';

const octokit = new (Octokit.plugin(createPullRequest))({
  auth: process.env.GITHUB_TOKEN,
});
```

Reply:
I think the suggested change looks cleaner! And LGTM about using the plugin.

Reply:
Done db4ed6e